docs/docs/commands.md

```diff
@@ -40,44 +41,46 @@ These commands require a staff role or the Manage Server (`MANAGE_GUILD`) permis
 | approve | Approve a submitted suggestion | [click](#approve-command) |
 | note | Add a note to a submitted suggestion | [click](#note-command) |
 | reject | Reject a submitted suggestion | [click](#reject-command) | 
-| staffsuggest | Submit a staff suggestion to be viewed by staff-only | [click](#staff-suggest-command) |
```

Comment:
That hyphen placement is appropriate.

Reply:
Whoops, I didn't clarify this correctly. Basically, it should still be `staff-only` and not `staff only` as that hyphen I originally had was an appropriate use. Other than that, I think this is all set unless @Skelmis has anything else he'd like to add on?